src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/PolicyTests.ps1

```diff
@@ -35,7 +35,7 @@ function Test-PolicyDefinitionCRUD
 
 	New-AzureRMPolicyDefinition -Name test2 -Policy "{""if"":{""source"":""action"",""equals"":""blah""},""then"":{""effect"":""deny""}}"
 	$list = Get-AzureRMPolicyDefinition
-	Assert-AreEqual 2 @($list).Count
+	Assert-AreEqual 83 @($list).Count
```

Comment:
>Assert-AreEqual 83 @($list).Count [](start = 1, length = 33)

this changes everytime the create is run,be sure it runs in playback mode also,if not revert it back